libr/socket/run.c

```diff
@@ -2,6 +2,7 @@
 
 /* this helper api is here because it depends on r_util and r_socket */
 /* we should find a better place for it. r_io? */
+#include <stdbool.h>
```

Comment:
that should be included already in r_types.h which comes from r_util. no need to include it here